docs/source/install/deb.rst

```diff
@@ -1,84 +1,210 @@
 Ubuntu / Debian
 =================
 
-|st2| debian packages have been tested and precompiled for Ubuntu 14.04.
+This guide provides step-by step instructions on installing StackStorm on a single box on a Ubuntu/Debian.
 
---------------
+.. contents::
+
+
+Minimal installation
+--------------------
+
+Install Dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Install MongoDB, RabbitMQ, and PostgreSQL.
+
+  .. code-block:: bash
+
+    sudo apt-get install -y mongodb-server rabbitmq-server postgresql
+
+
+Setup repositories
+~~~~~~~~~~~~~~~~~~~
+
+  .. code-block:: bash
+
+    wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
+    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
```

Comment:
@lakshmi-kannan Is it a plan to let users install from `staging` repos until creating Staging st2 box with all CI that deploys packages to normal production repos?